RileyCz.md

```diff
@@ -0,0 +1,30 @@
+
+---
+name: "@RileyCz"
+project: "Barometric Altimiter"
+---
+
+# Barometric Altimiter
+
+## Summary
+
+I'm planning on building a portable barometric altimiter which can either read a pressure altitude for a given barometric setting or display the current barometric setting. I was inspired to build this because I am currently in the process of becoming a pilot and I thought that this would be a cool way to expand my knowledge on coding and hardware whilst also paying tribute to my passion for aviation. This will be my first using hardware and I think it will be an exciting opportunity to learn new things.
+
+## Plan
+
+I will use a touchscreen LCD display as the main interface which allows the user to switch between reading an altitude or displaying the barometric setting. It will use a BMP280 I2C for barometric readings which will then be relayed onto the display.
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Arduino Rev 3    | [https://www.progressiveautomations.ca/products/lc-066?variant=19694939209815](https://www.progressiveautomations.ca/products/lc-066?variant=19694939209815)         | $28.99  |
+| Adafruit BMP280 I2C | [https://www.adafruit.com/product/2651](https://www.adafruit.com/product/2651)  | $25.45 |
+| Touchscreen LCD Display | [https://www.amazon.ca/480x320-Screen-Module-Arduino-Without/dp/B07NWH47PV/](https://www.amazon.ca/480x320-Screen-Module-Arduino-Without/dp/B07NWH47PV/)  | $21.25 |
+| Battery Holder x2 | [https://www.pcboard.ca/cr2032-battery-holder-with-switch](https://www.pcboard.ca/cr2032-battery-holder-with-switch)  | $14.53 |
+| CR2023 Battery | [https://www.homedepot.ca/product/duracell-2032-lithium-coin-battery-with-bitter-coating-4-pack/1001619397](https://www.homedepot.ca/product/duracell-2032-lithium-coin-battery-with-bitter-coating-4-pack/1001619397)  | $11.96 |
+| Soldering Iron | [https://www.amazon.ca/Soldering-Adjustable-Temperature-Electric-Repairing/dp/B0BBZB3G26/](https://www.amazon.ca/Soldering-Adjustable-Temperature-Electric-Repairing/dp/B0BBZB3G26/)  | $10.99 |
+| Solder Lead Free | [https://www.digikey.ca/en/products/detail/american-hakko-products,-inc./611-1/6228764](https://www.digikey.ca/en/products/detail/american-hakko-products,-inc./611-1/6228764)  | $29.04 |
+| Total           |                                       | $142.21 |
```

Comment:
Do you need a breadboard & wires for prototyping? Do you need any passives (resistors, capacitors, diodes, etc) for your circuit?